lib/apiservers/portlayer/restapi/handlers/scopes_handlers.go

```diff
@@ -366,7 +367,9 @@ func toScopeConfig(scope *network.Scope) *models.ScopeConfig {
 		pools = append(pools, p.String())
 	}
 
-	sc.IPAM = pools
```

Comment:
Is this change necessary? If so, how about sc.IPAM = append(sc.IPAM, pools...)

Reply:
Good catch! Thanks!